Sources/ArcGISToolkit/Components/Augmented Reality/CalibrationView.swift

```diff
@@ -0,0 +1,193 @@
+import ArcGIS
+
+extension WorldScaleGeoTrackingSceneView {
+    /// A view that allows the user to calibrate the heading of the scene view camera controller.
+    struct CalibrationView: View {
+        /// The view model for the calibration view.
+        @ObservedObject var viewModel: ViewModel
+        /// The scene camera controller heading.
+        @State private var heading: Double = 0
+        /// The scene camera controller elevation.
+        @State private var elevation: Double = 0
+        
+        var body: some View {
+            VStack {
+                HStack(alignment: .firstTextBaseline) {
+                    Text("Calibration")
+                        .font(.title)
+                        .lineLimit(1)
+                    Spacer()
+                    dismissButton
+                        .layoutPriority(1)
+                }
+                .padding(.bottom)
+                headingSlider
+                Divider()
+                elevationSlider
+            }
+            .padding()
+            .background(.regularMaterial)
+            .clipShape(RoundedRectangle(cornerRadius: 15))
+            .frame(maxWidth: 350)
+            .padding()
+            .task {
+                heading = viewModel.cameraController.originCamera.heading
+                elevation = viewModel.cameraController.originCamera.location.z ?? 0
+            }
+        }
+        
+        @ViewBuilder
+        var headingSlider: some View {
+            VStack {
+                HStack {
+                    Stepper() {
+                        HStack {
+                            Text("Heading")
+                                .font(.body.smallCaps())
+                                .foregroundStyle(.secondary)
+                            Spacer()
+                            Text((viewModel.calibrationHeading ?? viewModel.cameraController.originCamera.heading), format: .number.precision(.fractionLength(0)))
+                            Spacer()
+                        }
+                    } onIncrement: {
+                        rotateHeading(by: 1)
+                    } onDecrement: {
+                        rotateHeading(by: -1)
+                    }
+                }
+                JoystickSliderView()
+                    .onSliderValueChanged { delta in
+                        rotateHeading(by: delta)
+                    }
+            }
+        }
+        
+        @ViewBuilder
+        var elevationSlider: some View {
+            VStack {
+                HStack {
+                    Stepper() {
+                        HStack {
+                            Text("Elevation (m)")
+                                .font(.body.smallCaps())
+                                .foregroundStyle(.secondary)
+                            Spacer()
+                            Text((viewModel.calibrationElevation ?? viewModel.cameraController.originCamera.location.z ?? 0), format: .number.precision(.fractionLength(0)))
```

Comment:
The meters designation should go on the value, not the label.

```suggestion
                            Text("Elevation")
                                .font(.body.smallCaps())
                                .foregroundStyle(.secondary)
                            Spacer()
                            Text((viewModel.calibrationElevation ?? viewModel.cameraController.originCamera.location.z ?? 0), format: .number.precision(.fractionLength(0)))
                            + Text("m")
```

Reply:
Consider using a measurement format style, which is best practice for formatting measurements:

```swift
Text(
    Measurement<UnitLength>(value: elevationDelta, unit: .meters),
    format: .measurement(width: .narrow, numberFormatStyle: .number.precision(.fractionLength(0)))
)
```

If you want to, you could even store the elevation delta as a measurement.